add a showed event

bkrrr opened the pull request:
it is called after show and is super helpful for some visualizations, which
need a visible container to make a correct size computation. Like SVG/D3
implementations.

mattgodbolt commented:
Many thanks for this. I've merged (with tests added). I renamed 'showed' to 'shown' as that seemed to be more grammatically correct. Hope that's not going to cause issues!

bkrrr commented:
No really :). It`s from on naming convention for events, like "before,
during, after". No sure, in which language i picked that up.

- show
- showing
- showed

- delete
- deleting
- deleted

Anyways, cool. thank you for merging.


On Thu, Sep 22, 2016 at 2:52 AM, Matt Godbolt <notifications@github.com>
wrote:

> Many thanks for this. I've merged (with tests added). I renamed 'showed'
> to 'shown' as that seemed to be more grammatically correct. Hope that's not
> going to cause issues!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/deepstreamIO/golden-layout/pull/124#issuecomment-248784065>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANprRqdIrXTXXD1E-b54muzAkgn3qeVJks5qsdEwgaJpZM4JihuL>
> .
>
